docs/guides/getting-started-java-spring.md

```diff
@@ -133,9 +133,9 @@ See our documentation on [adding the Spring Zeebe SDK to your project](/apis-too
 </repositories>
 ```
 
-2. Add the following dependency your `pom.xml` file, as a child of the `<dependencies>` element:
+2. Add the following dependency to your `pom.xml` file, as a child of the `<dependencies>` element:
 
-:::warning
+:::caution
 Due to a bug in 8.6, use the Spring Zeebe SDK version `8.5.0`.
```

Comment:
```suggestion
The Spring Zeebe SDK is currently pinned to version `8.5.0`.
```

^ If we'd like to remove the bug language, we can just note that it's currently set at 8.5 (and therefore required).